### PR TITLE
feat: LT-1119 - Exporters can post notes on their applications

### PR DESCRIPTION
**Pull request by jfaracik**
- View case notes on applications
- Make case notes

### CODE REVIEW

File: cases/views.py [hunk 1]
```diff
@@ -75,14 +77,57 @@ def post(self, request, **kwargs):
...
+    def post(self, request, **kwargs):
+        case_id = str(kwargs['pk'])
+        response, status_code = post_case_notes(request, case_id, request.POST)
+
+        if status_code != 201:
+
+            errors = response.get('errors')
+            if errors.get('text'):
+                error = errors.get('text')[0]
+                error = error.replace('This field', 'Case note')
```
Comment (lukesa) [inline cases/views.py]: get content string

File: templates/cases/case/application-case.html [hunk 2]
```diff
@@ -107,6 +107,88 @@ <h3 class="govuk-heading-m">Denial Reasons</h3>
...
+	{% for country in data.case.application.destinations.data %}
+		<p class="govuk-label">
+			{{ country.name }}
+		</p>
+	{% endfor %}
+{% endif %}
+
+<br><br><br><br>
+
+<h3 class="govuk-heading-m">Goods Location</h3>
```
Comment (lukesa) [inline templates/cases/case/application-case.html]: get content string

File: templates/cases/case/tabs/activity.html [hunk 3]
```diff
@@ -1,135 +1,52 @@
...
-					</button>
+<form method="post">
+	{% csrf_token %}
+	<div class="lite-expandable-textarea lite-case-note-no-transition">
+		<textarea id="case_note" name="text" cols="80" class="govuk-textarea" placeholder="Add case note"></textarea>
+		<div class="lite-expandable-textarea-controls">
+			<div class="govuk-checkboxes govuk-checkboxes--small">
+				<div class="govuk-checkboxes__item">
+					<input class="govuk-checkboxes__input" type="checkbox" id="is_visible_to_exporter" name="is_visible_to_exporter" value="True">
+					<label class="govuk-label govuk-checkboxes__label" for="is_visible_to_exporter">Make visible to exporter</label>
```
Comment (lukesa) [inline templates/cases/case/tabs/activity.html]: get content string

File: templates/cases/case/tabs/activity.html [hunk 4]
```diff
@@ -1,135 +1,52 @@
...
+			<a id="case-note-cancel-button" href="{% url 'cases:case' data.case.id %}" class="govuk-button govuk-button--secondary" type="button" draggable="false">
+				Cancel
+			</a>
+			<button class="govuk-button" type="submit" id="button-post-note">
+				Post note
+			</button>
 		</div>
 	</div>
+	<p id="case_note-warning" class="govuk-hint govuk-character-count__message" aria-live="polite">
+		You can enter up to 2200 characters
```
Comment (lukesa) [inline templates/cases/case/tabs/activity.html]: get content string

File: cases/views.py (hunk 1 above)
Comment (cstobart) [reply cases/views.py]: This is a TODO and will be changed next time we touch the API stuff, apparently for some reason we can't do it in this story

**Pull request merged**